autorest/namer/name_converter.py

```diff
@@ -113,18 +113,19 @@ def _convert_language_default_python_case(schema, **kwargs):
             schema["language"]["python"]["summary"] = schema_summary
 
     @staticmethod
-    def _convert_language_default_pascal_case(schema):
+    def _convert_language_default_pascal_case(schema, is_object):
         if schema["language"].get("python"):
             return
         schema['language']['python'] = dict(schema['language']['default'])
 
         schema_description = schema["language"]["default"]["description"].strip()
-        if not schema_description:
-            # what is being used for empty ObjectSchema descriptions
-            schema_description = schema["language"]["python"]["name"]
-        if schema_description and schema_description[-1] != ".":
-            schema_description += "."
-        schema["language"]["python"]["description"] = schema_description
+        if is_object:
```

Comment:
Don't like the "is_object" here :(.

If I understand correctly, we try to create a fake description in the object has no description. why this can't be in "_convert_object_schema" just before calling "_convert_language_default_pascal_case"? It's not even conversion, so placing this code in "_convert_language_default_pascal_case" seems wrong to me.

Reply:
that makes a lot of sense, going to switch that. thanks!